Logger.Tests/Logger.Tests.csproj

```
@@ -1,9 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
 		<IsPackable>false</IsPackable>
-		<LangVersion>11.0</LangVersion>
+		<LangVersion>12</LangVersion>
```

```suggestion
    <LangVersion>12.0</LangVersion>
```
12.0 is the current lang version. Also making spacing consistent with rest of the file